cross/sshfs/digests

```diff
@@ -0,0 +1,6 @@
+sshfs-fuse-2.5.tar.gz SHA1 a28ed6c04847611816452441b14ad9a0bb6df842
+sshfs-fuse-2.5.tar.gz SHA256 e9171452e5d0150b9c6a2158fd2e2dcefb5d5d03ba4d208949e00a3a46c6e63e
+sshfs-fuse-2.5.tar.gz MD5 17494910db8383a366b1301e5f5148a9
+sshfs-fuse-2.5.tar.gz SHA1 a28ed6c04847611816452441b14ad9a0bb6df842
```

Comment:
Either recreate (`rm -rf digests && make digests`), or simply remove the last three entries.